### ISSUE TITLE
Modal bug

### ISSUE THREAD

**Issue by robertUzias** [patrickkunka/mixitup#401]
Hello. I've encounter a bug I don't know how to fix. Inside my "gallery cards", whenver I click a button to open a modal, it jumps to top of the page until I close it again.

How could I fix this?

**Comment by patrickkunka** [comment]
This doesn't sound like its related to MixItUp, but if you have a link to your project I can take a look.

**Comment by robertUzias** [comment]
Hello Patrick. I found the triggerer of it. I'm using this template: [TITAN](http://markup.themewagon.com/titan/index_finance.html), I don't have the project ready yet, but I found out this exactly line of code was the triggerer of the bug:

`html,
body {
    height: 100%;
    background: #f2f2f2;
}`

It's part of the original mixitup 3 css stylesheet. I don't really understand why it happens, but I already solved it. Thanks again, I really appreciate your help. Your plugin is awesome. Really.
